app/src/main/java/tech/pauly/findapet/discover/AnimalListAdapter.java

```diff
@@ -31,13 +31,7 @@ public int getItemCount() {
     }
 
     public void setAnimalItems(List<AnimalListItemViewModel> items) {
-        clearItems();
         animalItems.addAll(items);
         notifyDataSetChanged();
     }
-
-    public void clearItems() {
```

Comment:
Don't want to clear anymore, just keep adding to the list. Also no longer necessary because the lists are backed with different instances of VMs.